Cargo.toml

```diff
@@ -4,7 +4,7 @@ name = "http"
 # - Update html_root_url in lib.rs.
 # - Update CHANGELOG.md.
 # - Create git tag
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
I find it preferable to keep the version change to a separate commit, not part of any specific other feature or fix.

Reply:
I agree with you in general, that's why I did that in ba39dbb separately.  I included here because the PR really only makes sence in an 0.2.3.  Are you concerned about a squashed merge to master?

Reply:
If so, you can edit commits?

Reply:
Oh, I didn't notice there were multiple commits. The pattern so far I've used is to have a tagged commit with the version change when it's time to actually release.

Reply:
You could or I could reorder them?

Reply:
OK, I rebase it with what I think matches the pattern you've been using. Hope that helps?

Reply:
Note however, I didn't put in today as the release date in CHANGELOG.md.